### DISCUSSION TITLE
How do I use an AmbientLight?

### DISCUSSION THREAD

**Discussion by Newbytee** [bevyengine/bevy#6094] [Q&A] [answered]
Hi! I'm new to Bevy, and I'm playing around with a stripped-down version of the Alien Cake Addict example to understand Bevy better. I want to replace the `PointLightBundle` with an `AmbientLight`, but I don't quite understand how to do this. I tried using `spawn_bundle` to spawn in an `AmbientLight`, but this doesn't work as `AmbientLight` does not implement the trait `Bundle`. This leaves me wondering, how am I supposed to use AmbientLight given that there doesn't seem to be an AmbientLightBundle type?

**Comment by rparrett** [comment, marked as answer]
`AmbientLight` is a global property and is meant to be inserted as a resource. You can see an example of it in use here: https://bevyengine.org/examples/3d/lighting/
